docs/core/linux-prerequisites.md

```diff
@@ -114,11 +110,42 @@ To install .NET Core dependencies on RHEL Server:
     ```bash
     yum install scl-utils
     ```
-3. Install .NET Core 1.1 (and all dependencies):
+3. Install .NET Core
+# [.NET Core 2.x](#tab/netcore2x)
+
+Install .NET Core 2.0 SDK and Runtime:
+    ```bash
+    yum install rh-dotnet20
+    ```
+Enable .NET Core 2.0 SDK/Runtime for your environment:
+    ```bash
+    scl enable rh-dotnet20 bash
+    ```
+
+# [.NET Core 1.x](#tab/netcore1x)
+
+**.NET Core 1.1**
+
+Install .NET Core 1.1 SDK and Runtime:
     ```bash
     yum install rh-dotnetcore11
+    ```
+Enable .NET Core 1.1 SDK and Runtime for your environment:
+    ```bash
     scl enable rh-dotnetcore11 bash
     ```
+
+**.NET Core 1.0**
+
+Install .NET Core 1.0 SDK and Runtime:
+    ```bash
+    yum install rh-dotnetcore10
+    ```
+Enable .NET Core 1.0 SDK and Runtime for your environment:
+    ```bash
+    scl enable rh-dotnetcore10 bash
+    ```
+---
```

Comment:
Omair, This is great!
Just one set of changes and we are good to go.
The code snippet sections inside the tab (# [.NET Core 2.x](#tab/netcore2x)) need to be fully left justified to display correctly. Look at the tabs in the Ubuntu section for an example.
Please update those and I will get it merged. Thanks for your contribution!!!